docs/specs/clients/core/identity/identity-keys.md

```diff
@@ -33,24 +33,47 @@ Identity Keys are authorized by a blockchain account by the wallet user signing
 
 This results in a CACAO that authorizes the client's Identity Key to sign messages on the behalf of the blockchain account.
 
-For example:
+### Domain Authorization
+
+The statement will determine what domains the Identity Key is authorized to control via limited (only this domain) or unlimited access (all domains).
+
+#### Limited access
+
+Example [CAIP-122](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-122.md) for limited access:
 
 ```
 app.example.com wants you to sign in with your Ethereum account:
 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2
 
-TODO: statement
+I further authorize this DAPP to send and receive messages on my behalf for this domain and manage my identity at identity.walletconnect.com.
 
 URI: did:key:z6MkqJ6qV18zBazggzhGMHNgadEQGbX9RceEH3j2G6kNTbKq
 Version: 1
 Chain ID: 1
 Nonce: bb0b6514e8a5e817
 Issued At: 2022-12-09T15:29:36.509Z
+Resources:
+- https://identity.walletconnect.com
```

Comment:
Should we go with `https://identity.walletconnect.com` or `https://keys.walletconnect.com`?

Reply:
We currently use `keys`, let's stick with that no?

Reply:
I think so too